index.js

```diff
@@ -1,77 +1,87 @@
 'use strict';
 
-module.exports = {
-  parserOptions: {
-    ecmaVersion: 2022,
-  },
-  env: {
-    es6: true,
-  },
-  extends: [
-    'eslint:recommended',
-  ],
-  rules: {
-    'comma-dangle': ['error', 'always-multiline'],
-    'eol-last': 'error',
-    'eqeqeq': 'error',
-    'indent': ['error', 2, {
-      'MemberExpression': 1,
-      'SwitchCase': 1,
-    }],
-    'max-params': ['error', 3],
-    'no-constant-condition': 'off',
-    'no-empty': ['error', { allowEmptyCatch: true }],
-    'no-eval': 'error',
-    'no-restricted-globals': ['error', 'event'],
-    'object-shorthand': 'error',
-    'one-var': ['error', 'never'],
-    'quotes': ['error', 'single', {
-      'allowTemplateLiterals': true,
-      'avoidEscape': true,
-    }],
-    'require-await': 'error',
-    'semi': 'error',
+const js = require('@eslint/js');
```

Comment:
Since we are directly importing this, it should be a real dep and not a peerDep.

Reply:
Good point, made an update to the other repo, https://github.com/CrowdStrike/eslint-config-crowdstrike-node/pull/95